<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#IpAddress[state=&quot;0&quot;]{
	background-color: rgb(224, 27, 36);
}
#IpAddress[state=&quot;1&quot;]{
	background-color: rgb(87, 227, 137);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ip Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IpAddress">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>281</width>
       <height>28</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>000.000.000.000;_</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="state" stdset="0">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="Port">
     <property name="geometry">
      <rect>
       <x>161</x>
       <y>80</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>65536</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>106</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>256</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>510</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Send Data</string>
    </property>
    <widget class="QLineEdit" name="SendData">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Send">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>106</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
